client/src/pages/workspace/Workspace.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from "react";
+import Board from "./board/board";
+import {BoardEntity} from "../../model/entity/BoardEntity";
+import {List, ListItem} from "@mui/material";
+import AddBoard from "./board/AddBoard";
+
+const Workspace: React.FC = () => {
+    const [boards, setBoards] = useState<BoardEntity[]>([
+        {id: 1, name: 'white board', description: 'mega ultra super cool red board', createdAt: new Date(), updatedAt: new Date()},
+        {id: 2, name: 'board2', description: 'boardDesc2', createdAt: new Date(), updatedAt: new Date()}
+    ]);
```

Comment:
Right now there're predefined values in the useState. In the future, you are gonna send a request to the server to get these boards. To avoid making many changes in the future, let's make it extensible first.

1. Create a new folder `services` in the root
2. Create a file `board.ts`
3. create a function `getBoardsService()` which for now return your mock boards. In the future we will implement here requests to the server
4. Go back to Workspace component
5. Set initial values to empty array `useState<BoardEntity[]>([]);`
6. Create a function `getBoards()`. Inside call `getBoardsService()` and set new boards to your state
7. Now we need to somehow call `getBoards()` automatically when we open a page. Create useEffect hook https://react.dev/reference/react/useEffect and call getBoards() inside. UseEffect is used to automatically trigger some actions depending on some conditions/events.
```
useEffect(() => {
        getBoards();
    }, []);
```
`, []` second parameter tells react when rerun useEffect. `[]` - means run it on startup. You can do this way `[boards]` it means rerun useEffect whene boards are changed.
Have a look more about useEffect and try to test it with diffrent conditions.
Same shit will be for other cases where you expect get data from the server